test/inductor/test_torchinductor_opinfo.py

```diff
@@ -346,13 +360,24 @@ def process(device_type):
     # linalg._svd's return value has different strides on CUDA vs CPU which causes this
     # In test_meta.py there is a mechanism to skipping strides checks for some ops
     # (including _linalg_svd), possibly we should have something similar here
```

Comment:
The comment above should also be removed, right?

Reply:
Removed the comment, this was a mistake on my part.